### PR TITLE
Variants for the large figurine

### PR DESCRIPTION
**Pull request by Daved27hundred**
#### Summary
None

#### Purpose of change

The large figurine only has one variant, so it always shows up as the `t-rex figurine`. Now it has several variants.

#### Describe the solution

Created some variants in toy.json based on things you'd probably see being sold for 200 dollars at a Con. Also based them on some things I like. I kept the names vague to avoid any license issues.

#### Describe alternatives you've considered

If I need to do something else so they don't violate the license please let me know.

#### Additional context

N/A

### CODE REVIEW

**Comment by freepizzas-dev** [comment]
Love it. A small note:

line 114, "it's" should be "its", just like earlier in the sentence

File: data/json/items/toy.json [hunk 1]
```diff
@@ -101,6 +101,55 @@
         "name": { "str": "T-rex figurine" },
         "description": "A relatively large plastic figurine of a T-rex.",
         "weight": 1
+      },
+      {
+        "id": "toy_figure_mech",
+        "name": { "str": "Mech figurine" },
```
Comment (TheShadowFerret) [inline data/json/items/toy.json:107]:
```suggestion
        "name": { "str": "mech figurine" },
```

File: data/json/items/toy.json [hunk 2]
```diff
@@ -101,6 +101,55 @@
...
+        "description": "A large figurine of a humanoid robot, armed with a rifle, with a smaller figure sitting inside the head.",
+        "weight": 1
+      },
+      {
+        "id": "toy_figure_knight",
+        "name": { "str": "Knight figurine" },
+        "description": "A large figurine of a knight in shining armor standing watch, with its sword planted at the ground at it's feet.",
```
Comment (TheShadowFerret) [inline data/json/items/toy.json:114]:
```suggestion
        "name": { "str": "knight figurine" },
        "description": "A large figurine of a knight in shining armor standing watch, with its sword planted at the ground at its feet.",
```

File: data/json/items/toy.json [hunk 3]
```diff
@@ -101,6 +101,55 @@
...
+        "weight": 1
+      },
+      {
+        "id": "toy_figure_t800",
+        "name": { "str": "Robot figurine" },
```
Comment (TheShadowFerret) [inline data/json/items/toy.json:119]:
```suggestion
        "name": { "str": "robot figurine" },
```

File: data/json/items/toy.json [hunk 4]
```diff
@@ -101,6 +101,55 @@
...
+        "description": "A very intricate looking model of a robotic skeleton with red eyes and a futuristic weapon in its hand.",
+        "weight": 1
+      },
+      {
+        "id": "toy_figure_caiman",
+        "name": { "str": "Lizardman figurine" },
```
Comment (TheShadowFerret) [inline data/json/items/toy.json:125]:
```suggestion
        "name": { "str": "lizardman figurine" },
```

File: data/json/items/toy.json [hunk 5]
```diff
@@ -101,6 +101,55 @@
...
+        "description": "An expensive figurine of a buff man in streetwear with a lizard head, holding a pair of knives in his hands.",
+        "//": "yes this is a dorohedoro reference",
+        "weight": 1
+      },
+      {
+        "id": "toy_figure_anime",
+        "name": { "str": "Anime figurine" },
```
Comment (TheShadowFerret) [inline data/json/items/toy.json:132]:
```suggestion
        "name": { "str": "anime figurine" },
```

File: data/json/items/toy.json [hunk 6]
```diff
@@ -101,6 +101,55 @@
...
+        "description": "An expensive figurine of a cartoon woman in flashy clothing.",
+        "weight": 1
+      },
+      {
+        "id": "toy_figure_heavyweapons",
+        "name": { "str": "Video game figurine" },
```
Comment (TheShadowFerret) [inline data/json/items/toy.json:138]:
```suggestion
        "name": { "str": "video game figurine" },
```

File: data/json/items/toy.json [hunk 7]
```diff
@@ -101,6 +101,55 @@
...
+        "description": "A large figurine of a videogame character, holding a massive gatling gun in his giant arms.",
+        "weight": 1
+      },
+      {
+        "id": "toy_figurine_smoker",
+        "name": { "str": "Zombie figurine" },
```
Comment (TheShadowFerret) [inline data/json/items/toy.json:144]:
```suggestion
        "name": { "str": "zombie figurine" },
```

File: data/json/items/toy.json [hunk 8]
```diff
@@ -101,6 +101,55 @@
...
+        "description": "A nasty looking figurine of a mutant zombie, with a large tongue sprouting from its face.  You thankfully haven't encountered something like that yet.",
+        "weight": 1
+      },
+      {
+        "id": "toy_figure_alien",
+        "name": { "str": "Alien figurine" },
```
Comment (TheShadowFerret) [inline data/json/items/toy.json:150]:
```suggestion
        "name": { "str": "alien figurine" },
```
I would say "just write xenomorph here," but these do appear meant to be somewhat generic.

**Review by TheShadowFerret — COMMENTED** [review]
Non-proper nouns get da lowercase

**Comment by Karol1223** [comment]
Thank you. I hate writing variants so I ended up only doing one in hope someone else would pick up on it. You have saved us all. Mostly me, really, but shh.

File: data/json/items/toy.json [hunk 9]
```diff
@@ -101,6 +101,55 @@
...
+        "id": "toy_figure_t800",
+        "name": { "str": "robot figurine" },
+        "description": "A very intricate looking model of a robotic skeleton with red eyes and a futuristic weapon in its hand.",
+        "weight": 1
+      },
+      {
+        "id": "toy_figure_caiman",
+        "name": { "str": "lizardman figurine" },
+        "description": "An expensive figurine of a buff man in streetwear with a lizard head, holding a pair of knives in his hands.",
+        "//": "yes this is a dorohedoro reference",
```
Comment (Maleclypse) [inline data/json/items/toy.json:127]: I thought it was suburban commando.

**Pull request merged**